src/search-schema/data/definitions/enums/assay_types.yaml

```diff
@@ -169,24 +202,25 @@ salmon_rnaseq_bulk:
 #    alt-names: [salmon_rnaseq_bulk]
     alt-names: []
     primary: false
+    contains-pii: false
     vitessce-hints: []
 
 SNARE-ATACseq2:
     description: snATACseq (SNARE-seq2)
     alt-names: ['SNAREseq', 'SNARE2-ATACseq']
     primary: true
+    contains-pii: true
     vitessce-hints: []
 
 SNARE-RNAseq2:
     description: snRNAseq (SNARE-seq2)
     alt-names: ['SNARE2-RNAseq']
     primary: true
+    contains-pii: true
     vitessce-hints: []
 
 sc_atac_seq_snare_lab:
-#SNAREseq_lab:
     description: snATAC-seq (SNARE-seq2) [Lab Processed]
-#    alt-names: [sc_atac_seq_snare_lab]
     alt-names: []
     primary: false
     vitessce-hints: []
```

Comment:
@jswelling why don't the lab processed datasets have a `contains-pii` property?

Reply:
same goes for the one below...

Reply:
The lab processed datasets don't have it because I wasn't sure if we could trust them to *not* contain PII, so I left it to default to 'does contain PII'.  If we trust them to not include PII in the lab processed results, or if we do plan to actively scan for PII, we could set 'contains-pii' to False for the lab data.  Shall I do so?

Reply:
There is a lab processed seqfish that **does** have the explicit `contains-pii: false` label, so I was confused that some lab processed data had it and some didn't. I think we can set these lab processed data to false, and then implement our checking to be extra careful for future data sets. For the lab processed data we have, I think I already verified with them that there was no protected data.